cmd/goal/bundledGenesisInject.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by bundle_genesis_json.sh, along with langspec.json. DO NOT EDIT.
+// Code generated by bundle_genesis_json.sh. DO NOT EDIT.
```

Comment:
Given this change, shouldn't `cmd/goal/bundle_genesis_json.sh` also be part of the PR?

Reply:
Good question, I just ran `make generate` again to update the API specs and that's what the new generated file looked like. I think that this comment was written by a human deliberately, but the script is supposed to generate the comment without the `along with langspec.json`:

https://github.com/algorand/go-algorand/blob/cfe6e57874948dc50c51c8e8fa4e5c9d8b2c7317/cmd/goal/bundle_genesis_json.sh#L6

Related PR: https://github.com/algorand/go-algorand/pull/3639/files

Reply:
Ahhh.. I was reading the diff in the opposite direction... The PR is fixing the issue, not introducing it.

Reply:
(I'll reach out to @algorandskiy )

Reply:
Unresolving because I want to link this conversation to another issue.

Reply:
Linked this conversation in #3914

Reply:
this is my fault, I think I generated the file and then updated the sh file. Please proceed, this change looks good.